apps/charterafrica/src/payload/fields/donorsField.js

```diff
@@ -0,0 +1,21 @@
+import { deepmerge } from "@mui/utils";
+
+const field = {
+  name: "donors",
```

Comment:
We have donors for tools separately from organisations separately from people? I thought donors sponsored the work i.e. tools only.

Reply:
@kilemensi 
The explanation given,

A donor can donate to a tool but does not necessarily mean they are a donor for the organisation.
An organisation donor does not necessarily mean that they are donors for  all tools for that organisation that's why we had them separated.

Reply:
We do not have donors for contributors

Reply:
Still weird to think about a donor sponsoring an org directly and not specific work(s) but cool I guess.